ui/theme/partials/footer-scripts.hbs

```diff
@@ -1 +1,38 @@
 <script src="{{@root.site.ui.url}}/dist/index.js"></script>
+<style>
```

Comment:
Can this go in one of the `scss` files?

Reply:
@frangio Right,   let me add this to header.css.